### PR TITLE
Add PHP 5.6 features variadic `...` and exponential `**`

### PR DESCRIPTION
**Pull request by GromNaN**
Related to #46

Adding
- Variadic functions/Variable-length argument lists
- Exponentiation **

### CODE REVIEW

File: features.js [hunk 1]
```diff
@@ -759,10 +759,25 @@ const features = [
...
+        name: "Variadic functions via ...",
+        url: "https://www.php.net/manual/en/migration56.new-features.php#migration56.new-features.variadics",
+      },
+    ],
+  },
   {
     name: "Splat operator / argument unpacking (... operator)",
     description:
-      "The `...` operator, aka the splat operator or array unpacking operator",
+      "The <code>...</code> operator, aka the splat operator or array unpacking operator",
```
Comment (rosswintle) [inline features.js]: I'll leave this in, but note that, [as per the README](https://github.com/rosswintle/can-i-php#adding-php-features), backticks are converted to `<code>` tags.

Is there some reason you change these? Was this not working?

Comment (GromNaN) [reply features.js]: Oh, good to know. I noticed that both syntax were used and I naively thought that the backticks must be a mistake. Reverting this change.

File: features.js [hunk 2]
```diff
@@ -774,6 +789,20 @@ const features = [
       },
     ],
   },
+  {
+    name: "Exponential operators ** and **=",
+    description: "A right associative <code>** operator has been added to support exponentiation, along with a </code>**=</code> shorthand assignment operator.",
```
Comment (rosswintle) [inline features.js]: `<code>` tags aren't right here. I'll push a fix.

Comment (rosswintle) [reply features.js]: Looks like you already did this! 😄

File: features.js (hunk 1 above)
Comment (rosswintle) [reply features.js]: Our changes crossed. I think I've un-reverted it. 😆

**Comment by rosswintle** [comment]
Thanks, @GromNaN. Really appreciate the contribution. I'll get these merged!

**Comment by GromNaN** [comment]
With pleasure. You can tick 2 boxes in #46.

**Comment by rosswintle** [comment]
Ah yes. Really need to put some more time into this. Thanks again.

**Pull request merged**